pkg/kubelet/deviceplugin/endpoint.go

```diff
@@ -80,11 +74,9 @@ func (e *endpoint) getDevices() []*pluginapi.Device {
 // list initializes ListAndWatch gRPC call for the device plugin and gets the
 // initial list of the devices. Returns ListAndWatch gRPC stream on success.
 func (e *endpoint) list() (pluginapi.DevicePlugin_ListAndWatchClient, error) {
-	glog.V(3).Infof("Starting List")
```

Comment:
nit: leave this log here maybe useful for debug :-)

Reply:
This will get removed in https://github.com/kubernetes/kubernetes/pull/52149 anyways